email_app/app/safenet_comm/network.js

```diff
@@ -1,19 +1,24 @@
-import { shell } from 'electron';
+import { shell, remote } from 'electron';
 import { initializeApp, fromAuthURI } from '@maidsafe/safe-node-app';
 import { getAuthData, saveAuthData, clearAuthData,
   parseUrl, showError } from '../utils/app_utils';
 import pkg from '../../package.json';
 import { CONSTANTS } from '../constants';
 import 'babel-polyfill';
 
+const isDevMode = process.execPath.match(/[\\/]electron/);
+const app = remote.app;
 
+const cwd = process.cwd();
+const electronExt = process.platform === 'win32' ? '.cmd' : '';
 
 export const APP_INFO = {
   info: {
     id: pkg.identifier,
     scope: null,
     name: pkg.productName,
-    vendor: pkg.vendor
+    vendor: pkg.vendor,
+    customExecPath: isDevMode ? `${cwd}/node_modules/.bin/electron${electronExt} ${cwd}/app` : app.getPath('exe')
```

Comment:
I'm getting this error when I launch it: `Exec command must be an array of string arguments` which is due to the `customExecPath` not being an array as expected by safe-app-nodejs v0.6.1.